source/crud/tests/unified/bulkWrite-update-validation.yml

```diff
@@ -1,140 +1,73 @@
-# This file was created automatically using mongodb-spec-converter.
-# Please review the generated file, then remove this notice.
+description: "bulkWrite-update-validation"
+
+schemaVersion: "1.0"
 
-description: bulkWrite-update-validation
-schemaVersion: '1.1'
 createEntities:
-  -
-    client:
+  - client:
       id: &client0 client0
-      observeEvents:
-        - commandStartedEvent
-  -
-    database:
+      observeEvents: [ commandStartedEvent ]
+  - database:
       id: &database0 database0
-      client: client0
-      databaseName: &database_name crud-v2
-  -
-    collection:
+      client: *client0
+      databaseName: &database0Name crud-tests
+  - collection:
       id: &collection0 collection0
-      database: database0
-      collectionName: &collection_name crud-v2
-initialData:
-  -
-    collectionName: *collection_name
-    databaseName: *database_name
+      database: *database0
+      collectionName: &collection0Name coll0
+
+initialData: &initialData
+  - collectionName: *collection0Name
+    databaseName: *database0Name
     documents:
-      -
-        _id: 1
-        x: 11
-      -
-        _id: 2
-        x: 22
-      -
-        _id: 3
-        x: 33
+      - { _id: 1, x: 11 }
+      - { _id: 2, x: 22 }
+      - { _id: 3, x: 33 }
+
 tests:
-  -
-    description: 'BulkWrite replaceOne prohibits atomic modifiers'
+  - description: "BulkWrite replaceOne prohibits atomic modifiers"
     operations:
-      -
+      - name: bulkWrite
         object: *collection0
-        name: bulkWrite
         arguments:
           requests:
-            -
-              replaceOne:
-                filter:
-                  _id: 1
-                replacement:
-                  $set:
-                    x: 22
+            - replaceOne:
+                filter: { _id: 1 }
+                replacement: { $set: { x: 22 } }
         expectError:
-          isError: true
+          isClientError: true
```

Comment:
Switching from `isError` to `isClientError` is the only significant change for the `*-validation.yml` files.